### ISSUE TITLE
Internal Server Error 

### ISSUE THREAD

**Issue by barrybonsoir** [jonthornton/wp2tumblr#18]
Hi, I've exported my wordpress xml and when I attempt to upload I'm getting an "Internal Server Error."

" The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application."

Is this service still working? Thanks.

**Comment by barrybonsoir** [comment]
Nevermind. I figured this out. If the xml file is too big it doesn't work, so I am doing it in batches.

**Comment by jonthornton** [comment]
I'll look into raising the size limit, or at least provide an informative error message. Thanks for following up.

**Comment by barrybonsoir** [comment]
Thanks for the reply. So on closer inspection, I don't know if the size limit had anything to do with it. Rather, I might have one particular blog post that is somehow causing an error. I batched every month into separate xml files and there's one particular file that just won't load. I plan to research it more when I have time, or I might just add those posts manually. All the other posts worked.

I was led to believe it was a size issue, as I read somewhere else online that Tumblr had a ~70 post limit per day.

Thanks for creating this service. It's the only option I've found that works.
